### PR TITLE
Fix style in run_check.py

### PR DESCRIPTION
**Pull request by vdimir**
### Changelog category (leave one):

- Not for changelog (changelog entry is not required)

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
...

> Information about CI checks: https://clickhouse.tech/docs/en/development/continuous-integration/

### CODE REVIEW

**Review by Felixoid — APPROVED** [review]

**Comment by Felixoid** [comment]
Will merge right after style check

**Comment by vdimir** [comment]
https://s3.amazonaws.com/clickhouse-test-reports/36065/1322fab333d64516012ae25f9b3c19ab75d1a573/style_check__actions_.html

**Pull request merged**
